src/sentry/api/event_search.py

```diff
@@ -1483,7 +1487,6 @@ def resolve_field_list(fields, snuba_filter, auto_fields=True):
             columns.append("id")
         if not aggregations and "project.id" not in columns:
             columns.append("project.id")
-            project_key = PROJECT_NAME_ALIAS
```

Comment:
We need the project.name to generate links in the UI though.

Reply:
Oh, huh I think this was an accidental deletion resolving a conflict, I'll add it back in  :+1: